RELEASE-PLAN.md

```diff
@@ -1,8 +1,8 @@
 # Background and Context
 
-Releases of the Pathfinder Framework are strongly informing the tech specs releases.
+Releases of the PACT Framework are strongly informing the tech specs releases.
```

Comment:
The rebranding is from "Pathfinder Framework" to "PACT Methodology" - could you update all references to "Framework" as well?

Reply:
Check! Missed that one, should be good now. Also removed the last Supply Chain Actor (SCA) mention.